core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -1173,6 +1176,50 @@ public void quarter() {
     assertEquals(integerValue(4), eval(expression));
   }
 
+  private static Stream<Arguments> getTestDataForSecToTime() {
+    return Stream.of(
+        Arguments.of(1, "00:00:01"),
+        Arguments.of(2378, "00:39:38"),
+        Arguments.of(6897, "01:54:57")
+        //TODO: Discuss whether negative values should be supported like in MySQL.
```

Comment:
let's note this in the documentation that it's a limitation for the function

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/219/commits/9ae0057b2cc009e94c396c8713bc68d57020b1db